src/validation.h

```diff
@@ -230,6 +230,24 @@ struct PackageMempoolAcceptResult
         : m_tx_results{ {wtxid, result} } {}
 };
 
+/**
+ * Criteria for bypassing mempool checks.
+ */
+struct MemPoolBypass {
+    //! When true, run validation checks but don't submit to mempool.
+    bool m_test_accept{false};
+
+    //! When true, don't enforce mempool fee and capacity limits.
+    //! This is used to indicate that mempool limiting based on feerate should be bypassed.
+    //! This does not prevent mempool from accepting the transaction.
+    bool m_bypass_limits{false};
+
+    MemPoolBypass() {}
+
+    MemPoolBypass(bool test_accept, bool bypass_limits) :
```

Comment:
Prefer limiting this to designated initializers, or otherwise explicit assignment.

Reply:
I originally used only C++ 20 designated initializers in this PR, but it crashes on one (and only one) CI check (error C7555 on a Windows machine, if I remember correctly).

Something like:
`error C7555: use of designated initializers requires at least '/std:c++20'`

Reply:
We now use C++20 on that CI, so they should work fine.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25577/commits/30555e7608ed4e7bb6f5dc6d0eddd1722119e6e1.